examples/slackbot/Notebook.ipynb

```diff
@@ -141,17 +142,15 @@
     "\n",
     "# Format for the OpenAI API\n",
     "def format_prompt(prompt):\n",
-    "    return [f\"{p.user} @ {p.ts} ---> {p.text}\" for p in prompt].join(\"\\n\\n\") + \"\\n\\n###\\n\\n\"\n",
+    "    return \"start -> \" + \"\\n\\n\".join([f' {msg.user} --> {msg.text} ' for msg in prompt]) + \"\\n\\n###\\n\\n\"\n",
```

Comment:
python joins are the opposite of what you would expect.  `"string".join([list])`